nss-day-cohort-39/github-workflow-aquarium-cohort-39-mandarin-fish
# Added Finley the betta fish

###Changes that were made
-Added Finley the betta to index.html
-Added dialog button functionality to betta fish

###Steps to test
1. Verify that Finley the betta fish shows up in fish list section
2. Click Finley the betta fish dialog button to verify that the dialog for betta fish appears and has all of the details for betta fish.
